codelab/02_kpt-deploy/tutorial.md

```diff
@@ -0,0 +1,240 @@
+# Deploy Your Resource via `kpt` 
+
+<walkthrough-disable-features toc></walkthrough-disable-features>
+
+## Introduction
+
+
+#### What is kpt?
+
+Kpt is [an OSS tool](https://github.com/GoogleContainerTools/kpt) for Kubernetes packaging, which uses a standard format to bundle, publish, customize, update, and apply configuration manifests.
+
+#### What is kpt?
+
+Kpt is [an OSS tool](https://github.com/GoogleContainerTools/kpt) for Kubernetes packaging, which uses a standard format to bundle, publish, customize, update, and apply configuration manifests.
+
+#### What kpt can help you with
+    
+*  You will get an hands-on off-the-shelf experience about the **GitOps** CI/CD workflow in skaffold.
+*  You can validate each of your config changes **declaratively**.
+*  You **won't** encounter **version conflict** if the config hydration (a.k.a kustomize) mismatch with the deployment tool (e.g. kubectl). 
+*  You can prune your resources accurately with [a three-way merge strategy](https://kubectl.docs.kubernetes.io/pages/app_management/field_merge_semantics.html). 
```

Comment:
This link 404's when I try to access it

Reply:
👍 yeah, don't know why they are changed. I also checked the other links. They should all work now.